lesson14/css/styles.css

```diff
@@ -0,0 +1,122 @@
+/*reset*/
+*,
+*::before,
+*::after {
+	box-sizing: border-box;
+}
+
+html,
+body {
+	margin: 0;
+	padding: 0;
+}
```

Comment:
nits
It's fine, but reset.css wants to separate the files

Reply:
Thank you for your review.
I'm sorry that our response was delayed. 

I have separated it into style and reset CSS files.

[e43143c](https://github.com/nyattsu72/hanson-frontend-cource/pull/19/commits/e43143cc2388e923fdfa2ab1de5720e03ee32787)

Reply:
Thank you for your review.
I'm sorry that our response was delayed.

I have separated the style and reset CSS.

[e43143c](https://github.com/nyattsu72/hanson-frontend-cource/pull/19/commits/e43143cc2388e923fdfa2ab1de5720e03ee32787)